### ISSUE TITLE
Publications list -- link entries to full bib record in blacklight.

### ISSUE THREAD

**Issue by KraigBinkowski** [ycba-cia/blacklight-collections2#211]
Within the publications list display -- can these individual entries be made to link directly to the full bib records in Blacklight?

**Comment by flapka** [comment]
@KraigBinkowski I like this suggestion. When RBM revises its procedures for publications information (to mimic what's in place already for art collections) this linking functionality would be desirable for us too.

**Comment by yulgit1** [comment]
Does TMS have bibids that can be ported into lido and indexed along with each citation?

Or is there another way to make this link?  It is somewhat possible using solr, but not advisable, as would quickly drag down performance, and would be difficult to formulate a search that finds 1 and only 1 associated record.

**Comment by flapka** [comment]
@yulgit1 @KraigBinkowski Speaking with only a novice understanding TMS Bibliography module: I echo Eric's suggestion. It seems like it'd be really useful to have the Orbis bib ID as part of the data in TMS. Could it be recorded as an Alternate Number, of type YaleBibID (it won't be Orbis forever). With that piece of information expressed (in the LIDO), it seems like it should trivial to make the links.

If RB revises its practice to mimic the art collections publications format, I think we'd want to rely on (put in MARC) the Orbis ID to make such connections.

**Comment by KraigBinkowski** [comment]
right now the bibids are not part of the information added to the TMS bib module. brings up a couple of questions - how does this look for articles vs books and is there a way to mass import bibIDs into the bib module without adding them one by one manually (for existing records)?

**Comment by flapka** [comment]
On the mass import, that seems doable. If TMS could export a table for the Alternate numbers, I think we could do a *join* query with a table for items in REF, using the call number as the linking property, adding the bib ID to the TMS table.

I might even be able to do this myself, if the TMS table could be provided.

**Comment by edgartdata** [comment]
@flapka @KraigBinkowski Alternatively, it would be very easy to export bibliographic records IDs out of TMS. The reason I am mentioning this is that I don't think it matters where the joint between TMS and MARC is made, right? Otherwise, using Alternate Number in the TMS Bib Module should work, although we would need to check with David that this could be a bulk ingest.

**Comment by edgartdata** [comment]
@yulgit1 @KraigBinkowski @flapka returning to this issue as it is now in scope for LUX PoC (you are welcome!). 
From the convo above, is the next step for me to check with David Parsell if it is possible to bulk ingest OrbisBibIDs in TMS predicated on the titles of the citations? Titles might not be distinct enough for matches. Then what other identifiers/metadata elements could we use?

**Comment by flapka** [comment]
I 2nd Kraig's suggestion. Something like:
* Step 1: Attempt to execute a union query (TMS bib module / Orbis data) based on call numbers in each data set.
* Step 2: Manually review and edit data where the union query was not successful
* Step 3: Re-ingest data to TMS with Orbis bib IDs!

I'd be happy to provide a table of Ref's Orbis data, if that'd be helpful. Do citations also reference Orbis records outside of Ref?

**Comment by KraigBinkowski** [comment]
A small subset of records are non Ref Orbis records -- probably less than 5%. These would be Haas and/or Sterling Lib. -- Perhaps easiest to segregate these and add bibiIDs manually?

**Comment by flapka** [comment]
@KraigBinkowski  Yes, that sounds reasonable. Especially as a good working from home project, perhaps.

**Comment by edgartdata** [comment]
This is the entire list of bib citations in TMS (at least the ones that have a title) and 3,894 out of these 4,060 have a LC Call Number. TMS has other types of identifiers for some of these items: 

- Call Number (is that different from LC Call Number?)
- Class Number
- Control Number
- Dewey Number
- Government Number
- ISNB
- ISSN 
- LC Call Number
- LC Control Number
- OCLC Number

Unfortunately the TMS list view mode that exports to Excel only grabs the first identifier listed in the TMS Alternate Number field. I can generate smaller groups per number type if that's helpful though and see if there is a way to get all helpful identifiers into one document for this data reconciliation task.

[TMS Bib records September 3 2020.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/5169547/TMS.Bib.records.September.3.2020.xlsx)

**Comment by yulgit1** [comment]
Could the bibid be exposed in the LIDO something like:

<lido:event>
     <lido:eventID lido:type="TMS">34</lido:eventID>
     <lido:eventID lido:source="voyager">4226255</lido:eventID>

**Comment by flapka** [comment]
@KraigBinkowski @yulgit1 Reviewing the table that @edgartdata created and comparing it to a table of Ref library data reveals a couple of points:

1. There are subtle differences in call number syntax that are likely to make automated joins on call number unsuccessful. For example: TMS has "N12 W35 +A1 OVERSIZE  (YCBA)" where Voyager has "N12.W35 A1 (LC) Oversize)" for [this item](https://collections.britishart.yale.edu/catalog/orbis:10447558). 
2. On the other hand, it looks like the number of unique Orbis titles represented in TMS citations is much smaller than one might think. I did a grouping represented in the Excel file loaded herewith. If I crunched correctly, it appears that (from the 4000+ citations) there are only 115 unique ReferenceIDs and 100 unique call numbers. Unless I've made a mistake in the crunching, it looks a manual addition of Voyager IDs to this table (of 115 entries) might do the trick?

[citations_grouped.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/5170022/citations_grouped.xlsx)

**Comment by KraigBinkowski** [comment]
most of those identifiers are only present in the records we imported to TMS via Z39.50 protocol. We stopped doing that quite a while ago so they are only present in a smaller portion of the citations.  All of the actual call number data should be in the "LC call number" field -- if data is in the "Call Number" field its probably a mistake.

**Comment by KraigBinkowski** [comment]
If what you are saying is that there are only 115 unique books represented by the 4000 citations than I would say there has to be a mistake. I don't have the exact numbers at hand but that really can't be the case as we have been adding books and articles for over 10 years. The vast majority of these additions are small numbers of citations taken from books and not whole books containing a large amount of citations. If we can run a report on the titles in the bib module, that would give us an accurate count.

**Comment by flapka** [comment]
@KraigBinkowski I too found the idea of only 115 unique entries surprising, so I wonder if something is amiss. 

The spreadsheet that @edgartdata shared has 4060 rows, of which 93% come from just four (!) titles:


ReferenceID | Title | Alternate Number | CountOfAlternate   Number
-- | -- | -- | --
323 | A Concise Catalogue   of Paintings in the Yale Center for British Art | N590.2 A83 (YCBA) | 3021
1143 | A Catalogue of the   Paintings, Drawings, and Etchings of Philip Mercier | N12 W35 +A1 OVERSIZE    (YCBA) | 424
3977 | A Decade of Gifts and   Acquisitions | V2724 (YCBA) | 182
1732 | A biographical   dictionary of sculptors in Britain, 1660-1851 | NB496 B56 2009  (YCBA)   (Wall Shelf 3) | 171

Is it possible that the TMS export didn't snag all the entries?

**Comment by KraigBinkowski** [comment]
Yeah something is amiss - from our latest statistics (last annual report)  in this one year, staff added 513 citations and 441 book and article titles to TMS,  I wrote that this "ongoing effort has added over 14,000 bibliographic citations to  4,382 objects in the Center’s collection since the project began in 2009. "  So, the number should be closer to 14000 citations than 4000.

**Comment by edgartdata** [comment]
I think you are right. I did not get all the TMS bib records out because all searches I do not on other fields return the same number of results, i.e. 4,060. Let me work on this and get back to you.

**Comment by edgartdata** [comment]
@KraigBinkowski @flapka 

An update on this issue: David Parsell checked the TMS SQL tables and he finds the same number of TMS Bibliography Module records as I do, i.e. 4,060. We wondered for a moment if TMS might have "lost" records but a copy of the TMS database from May 4 2020 also has 4060 records so I think we can be pretty confident that this number is accurate.

What I think happened is that the TMS export mechanism into Excel had a problem. It seems that the process timed out somehow and the Excel list got populated with many duplicates records instead of real records. Unfortunately this export process is still not working today.

I have asked David for a list (from SQL) of all bib records that have LC Call Numbers and another one with all Bib Module records.

**Comment by edgartdata** [comment]
@edgartdata to identify the TMS Bib records that do not have LC Call Numbers.

**Comment by edgartdata** [comment]
@KraigBinkowski @flapka I have checked all the reports that we currently have available in the TMS Bib Module to export bib citations out of TMS and all the reports contains many duplicate unfortunately.

To get this moving, I queried TMS for Bib records that have a title (I figured 99% of TMS bibliographic citation records have a title), exported the results to Excel via a TMS report and then took out all the duplicated rows out of the Excel (based on the TMS ReferenceID). The result is this list of 4,832 TMS Bib Module records with 3 columns: TMS Bib Module ReferenceID, Title, and LC Call Number:

[YCBA TMS Bib Module records with titles 1-29-2021.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/5895069/YCBA.TMS.Bib.Module.records.with.titles.1-29-2021.xlsx)

I realize that my list might not be totally accurate in terms of number of Bib records but I figured it is close enough to get us started.

As we talked about, I hope this list helps you match the TMS Bib Module records in this list with their Orbis BibID, so that we can make actionable the titles that appear in the Publications section of RBM or P&D/P&S BlackLight records and have these titles link to their corresponding more complete Reference Library BL records.

**Comment by KraigBinkowski** [comment]
Thanks Emmanuelle - this number sounds like it could be close to the mark - given that many titles have several different objects cited. Francis and I can take a look and see if your report gives us what we need to be able to come up with a matching plan. Is it possible to include more fields to the spread sheet if we need them (thinking of author)?

Thanks very much for compiling this,

Kraig

Kraig Binkowski
Chief Librarian
Reference Library and Archives

Yale Center for British Art

1080 Chapel Street

P.O. Box 208280

New Haven, CT 06520-8280

+1 203 432 2846 | +1 203 415 1192 mobile


britishart.yale.edu<http://britishart.yale.edu/>

Explore the Center's exhibitions<https://britishart.yale.edu/exhibitions> online.
Connect on Facebook<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.facebook.com%2FYaleBritishArt%2F&data=02%7C01%7Clyn.rose%40yale.edu%7C02164475b8f54640e50d08d79f5be394%7Cdd8cbebb21394df8b4114e3e87abeb5c%7C0%7C0%7C637153089312502268&sdata=cWFWX9dZKjaC9SiTfDHkWQmsW0fv5Y%2B9aaSfG%2FWSo%2BA%3D&reserved=0>, Twitter<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Ftwitter.com%2FYaleBritishArt&data=02%7C01%7Clyn.rose%40yale.edu%7C02164475b8f54640e50d08d79f5be394%7Cdd8cbebb21394df8b4114e3e87abeb5c%7C0%7C0%7C637153089312512260&sdata=IFyktJi9i3QXR94LWitH5svX0D%2F9pQ8Bb4Vjjm8BWGI%3D&reserved=0>, Instagram<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.instagram.com%2Fyalebritishart%2F&data=02%7C01%7Clyn.rose%40yale.edu%7C02164475b8f54640e50d08d79f5be394%7Cdd8cbebb21394df8b4114e3e87abeb5c%7C0%7C0%7C637153089312512260&sdata=7hZ4%2BD6l1XUl13qTvjpB56lQ8hVH4VvnrhhR3lnKG4w%3D&reserved=0>, and YouTube<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.youtube.com%2Fyalebritishart&data=02%7C01%7Clyn.rose%40yale.edu%7C02164475b8f54640e50d08d79f5be394%7Cdd8cbebb21394df8b4114e3e87abeb5c%7C0%7C0%7C637153089312522255&sdata=RQmjS7zAcuAEdlGLt55LVMHp7gds5pqmys3g2PPxlWk%3D&reserved=0> @yalebritishart #YCBA

**Comment by KraigBinkowski** [comment]
Thanks Emmanuelle – this number sounds like it could be close to the mark – given that many titles have several different objects cited. Francis and I can take a look and see if your report gives us what we need to be able to come up with a matching plan. Is it possible to include more fields to the spread sheet if we need them (thinking of author)?

**Comment by flapka** [comment]
With the hope of doing semi-automated matching, is it possible to export a table that includes the ISBN numbers and the "Control Number" whenever a record includes these numbers? Matching on call number can be tricky, because of variations in syntax. ISBN numbers and control numbers (which are equal to BIB ID?), where present, might have fewer issues of complicated syntax.

**Comment by yulgit1** [comment]
As a TMS learning exercise I took a stab at this.  The number of objects is a little different 3832 in EDG's vs 4146.  @flapka let me know if this has all necessary and correct fields.


[bib-report-erj-jan29-2021.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/5897250/bib-report-erj-jan29-2021.xlsx)

**Comment by KraigBinkowski** [comment]
Thanks Eric - this looks great. I wonder what accounts for the 300 or so difference in titles? Francis, can we meet next week and come up with a plan for fields to use to automate the process of importing the bibids? This looks like a promising start.

**Comment by flapka** [comment]
@yulgit1 For some of the items in your report, the value for ISBN looks corrupted. For example (the third column): 

1298 | Gardens and cultural change | 9.7808840233e+12 | SB470.7 .G37 2007 (YCBA) | NULL | 8152450
-- | -- | -- | -- | -- | --

In TMS that ISBN value is 9780884023302. Is it corrupted by a data type version? Regardless, could you generate the report again in a way that addresses this issue? Thanks!

**Comment by edgartdata** [comment]
@flapka @KraigBinkowski @yulgit1 Could it be that the ISBN value for _Gardens and cultural change_ is corrupted is because that title is not linked to any TMS object records?
![image](https://user-images.githubusercontent.com/4093644/106911835-968d3780-66d0-11eb-802c-350c09db4e88.png)

**Comment by yulgit1** [comment]
Not corrupted, excel is automatically converting long numbers into scientific notation.  

To fix:

Right click on ISBN column header, choose "Format cells", "Custom" and choose the option that says 0 (second option below General).

**Comment by KraigBinkowski** [comment]
That is odd that the bib entry has no linked object. I am hopeful that it is an anomaly. I will check the book when I am next in the library.

**Comment by flapka** [comment]
> Right click on ISBN column header, choose "Format cells", "Custom" and choose the option that says 0 (second option below General).

Brilliant. Thanks @yulgit1 !

**Comment by flapka** [comment]
@KraigBinkowski As we previously discussed, I've started batch processing to match (and fill in) TMS publications data based on what's available in Voyager records. (The work is undertaken in tables outside of TMS.) A couple of questions:

1. Where we fill in OCLC numbers currently absent in TMS, would you like us to remove the non-numeric prefixes, such as "(OCoLC)" and "ocm"? I think they've been removed in the existing TMS data, and my sense is that this removal is advantageous.

2. For ISBN numbers:
a. When there are two numbers, because the MARC includes both the 13-digit and 10-digit forms (e.g. "9781416576495 (hardcover);1416576495 (hardcover)"), would you like to retain both? Or just the 13-digit version?
b. Do you want to retain the non-numeric text, e.g. "(hardcover)"?

**Comment by KraigBinkowski** [comment]
1. yes, please strip all prefixes
2. We just need one ISBN - just the 13 digit one (unless there is only a 10-digit number). No, we don't need any text such as "hardcover" if you can eliminate it.
Thank you,

**Comment by flapka** [comment]
@yulgit1 @KraigBinkowski @edgartdata 

Based on some semi-automated matching + manual look-ups by Kraig and his team, we now have a table ready to re-import back to the TMS publications module. The goal (as noted above) is to introduce a much more comprehensive and complete set of ILS and OCLC numbers. The table has three columns:

* ReferenceID (TMS bib record ID): this ID can be used to match data in the table to existing entries in TMS
* oclc_replaceToTMS: numbers in this column should replace data in TMS where Altnum = OCLC Number
* ILSno_addToTMS: numbers in this column should be added to TMS where Altnum = ILS Number (in theory these should replace existing numbers too, but in practice few if any of the records in this set are likely to have data in ILS Number)

The full table has 3743 records to update. For the purposes of testing the *process*, Kraig and I wonder if it would be useful to import/overlay a smaller set, such as the 20 also attached here. Would that make sense?

[Test list TMS Bib oclc ils numbers.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/8789163/Test.list.TMS.Bib.oclc.ils.numbers.xlsx)
[Full list TMS Bib oclc ils numbers.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/8789167/Full.list.TMS.Bib.oclc.ils.numbers.xlsx)

**Comment by KraigBinkowski** [comment]
@edgartdata The work that Eric did on the test list above just added the OCLC number and ILS number to the bib record for _This Other Eden (_TMSBib45). The linking of the bib entry to the full record is the  second part of the equation after we know the fields have been added correctly.

**Comment by KraigBinkowski** [comment]
@edgartdata @flapka we haven't yet asked Eric to create the links or what part of the bib entry to hot link. It is my opinion that if possible, it should only be our _Call Number_ field in the bibliography entry that links to the full record -- a bib entry in which the full entry is hotlinked seems visually distracting and perhaps confusing as online resources in the bibliography are hotlinked in this way. This is all up for discussion.

**Comment by yulgit1** [comment]
One thing I noticed in Dort just sampling the first publication, see below. It's ILS number https://hdl.handle.net/10079/bibid/584227, matches the call number. But the LIDO appellationvalue is nothing like what's in the bib record.  Maybe the citation is just a part of the whole?

```
<lido:event>
<lido:eventID lido:type="TMS">34</lido:eventID>
<lido:eventID lido:type="local" lido:source="ILS Number">584227</lido:eventID>
<lido:eventID lido:type="local" lido:source="LC Call Number">NJ18 T85 T87+ (YCBA)</lido:eventID>
<lido:eventID lido:type="local" lido:source="OCLC Number">07605713</lido:eventID>
<lido:eventID lido:type="local" lido:source="YCBA TMS Bibliographic Module record referenceID">4800</lido:eventID>
<lido:eventType>
<lido:conceptID lido:source="AAT" lido:type="local">300054686</lido:conceptID>
<lido:term>publication event</lido:term>
</lido:eventType>
<lido:eventName>
<lido:appellationValue>
^'Splashers,' Scrawlers' and 'Plasterers' : British landscape painting and the language of criticism, 1800-40^, Mallord Press, London, Summer 1990, pp. 5-11, NJ18 T85 T87+ (YCBA)
```

**Comment by edgartdata** [comment]
@KraigBinkowski https://hdl.handle.net/10079/bibid/584227 has the title "Turner Studies" in Orbis but our TMS citation with the same call number is for "'Splashers,' Scrawlers' and 'Plasterers' : British landscape painting and the language of criticism, 1800-40, 
Mallord Press, London, Summer 1990, pp. 5-11, NJ18 T85 T87+ (YCBA)"
Could it be a mistake with the call number in TMS? Or that this citation is for an article in "Turner Studies"?

**Comment by edgartdata** [comment]
@yulgit1 Knowing that we will discuss this tomorrow, I wonder if this option is viable:
- take out the callnumber out of the long citation in LIDO
- index the callnumber separately and append it to the long LIDO citation so that it is the only hyperlink element in the citation and it is there only once

**Comment by KraigBinkowski** [comment]
"@KraigBinkowski https://hdl.handle.net/10079/bibid/584227 has the title "Turner Studies" in Orbis but our TMS citation with the same call number is for "'Splashers,' Scrawlers' and 'Plasterers' : British landscape painting and the language of criticism, 1800-40, Mallord Press, London, Summer 1990, pp. 5-11, NJ18 T85 T87+ (YCBA)"
Could it be a mistake with the call number in TMS? Or that this citation is for an article in "Turner Studies"? "

**I think this is indeed a cataloguing mistake - I will check fully and correct it.**

**Comment by edgartdata** [comment]
There are 3 types of hyperlinks we would like to add to the bib citations:

1. A link from a YCBA P&D or P&S Blacklight record to a YCBA Ref Library or YCBA RBM Blacklight record
2. a link from YCBA P&D, P&S, Ref Library or RBM Blacklight record to a Yale Orbis/Quick Search record
3. A link to an external web resource, such as the Tate article https://www.tate.org.uk/research/in-focus/the-dolls-house-william-rothenstein/narrative-and-anti-narrative for https://collections.britishart.yale.edu/catalog/tms:5019

For 1. the plan is to use the ILS Number that points to a YCBA resource and then return the Blacklight URI for the resource, and hyperlink the LC Call Number with the BL URI. The LC Call Number would need to be taken out of the bib citation (we need to ask David to do that after we test our process for a couple of objects).

<lido:objectWebResource/>
<lido:objectID lido:type="local" lido:source="ILS Number">7838392</lido:objectID>
<lido:objectID lido:type="local" lido:source="LC Call Number">N5220 M552 P38 2007 OVERSIZE (YCBA)</lido:objectID>

Questions: can we do that for all bib citations that have a corresponding item in either Ref Lib or RBM? Is there a risk to loose some LC Call Numbers out of the bib citation because there would not be corresponding Ref Lib or RBM resources in Blacklight?

For 2. same process but point to Yale Orbis/Quick Search URIs. 
Questions: do we prefer Orbis? or Quick Search? @KraigBinkowski Do you have an example?

For 3. I think it would be helpful if we would create a new TMS AltNum Description and lido:source type for resources available online.  For this use case, we would need to ask David to take out the TMS Remarks field out of the bib citation concatenation.

<lido:eventID lido:type="TMS">5019</lido:eventID>
<lido:eventID lido:type="local" lido:source="LC Call Number">Available Online</lido:eventID>
<lido:eventID lido:type="local" lido:source="YCBA TMS Bibliographic Module record referenceID">3622</lido:eventID>
Question: @KraigBinkowski Are all citations marked as available online refer to non Yale resources? 

Feel free to poke holes in this plan!

**Comment by yulgit1** [comment]
"Could the logic say if we have an ILS no and the call number includes "YCBA", then make a YCBA link; otherwise don't?"

The answer is yes by indexing the callnumber as part of the ordered linked_citation_ss array.

**Comment by edgartdata** [comment]
@KraigBinkowski @flapka Also the Hyman link to the YCBA copy not working because there is no such copy, i.e. it is a film that has been classified as LC Call Number for efficiency? 
Kraig and I have already started talking about adding new types in TMS for such resources and it seems that it has now become a priority.

**Comment by edgartdata** [comment]
@KraigBinkowski Are all citations that have YCBA actually YCBA resources for which we can mint a link? I think that's @flapka's question? Francis correct me if I am wrong.

**Comment by edgartdata** [comment]
We are pushing this to production for next week.

We reviewed the risks and decided that the worst case scenario would be when "YCBA" is in the LC Call Number when that resource is NOT a YCBA resource but there is no way for us to know that ahead of time/to validate the links.

**Comment by flapka** [comment]
@edgartdata asked:
> So I guess the next question is: do we want to offer non YCBA BL links in our online bibliography? For example, if there are no YCBA copies of the book in our Ref Lib or RBM, but there is a copy at YUL, do we want to vend this link to our citation since Kraig's team have recorded it in TMS?

From previous discussions with Kraig, I believe it has been the intent to link to Orbis when YUL has a copy while YCBA does not. I think that'd be a user-friendly outcome. 

@edgartdata asked:
> @KraigBinkowski Are all citations that have YCBA actually YCBA resources for which we can mint a link? I think that's @flapka's question? Francis correct me if I am wrong.

Yes, I believe the presence of "YCBA" in the call number part of the citation is a uniformly reliable indicator that it is actually YCBA resource and that we can mint a Blacklight link for it.

**Comment by edgartdata** [comment]
This is great. Thanks Eric.

**Comment by flapka** [comment]
[bib_module_for_re-ingest.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/9869982/bib_module_for_re-ingest.xlsx)

@yulgit1 @KraigBinkowski The Excel file attached has updated ILS numbers and OCLC numbers, as provided by Kraig and co. Matching records in TMS using the referenceID (columnA), we'd like to add/replace OCLC numbers (column G) and ILS numbers (column H), leaving all the other bib module data unchanged (ignore the remaining columns in the Excel spreadsheet). 

Some entries will still lack an OCLC number, and that is okay. Go ahead and update the ILS number.

**Comment by KraigBinkowski** [comment]
> @KraigBinkowski https://hdl.handle.net/10079/bibid/584227 has the title "Turner Studies" in Orbis but our TMS citation with the same call number is for "'Splashers,' Scrawlers' and 'Plasterers' : British landscape painting and the language of criticism, 1800-40,
> Mallord Press, London, Summer 1990, pp. 5-11, NJ18 T85 T87+ (YCBA)"
> Could it be a mistake with the call number in TMS? Or that this citation is for an article in "Turner Studies"?

Finally circling back to this -- Bib record was indeed in error - The Journal title (Tuner Studies) and volume information was missing - I added both.
